enterprise/frontend/src/metabase-enterprise/whitelabel/components/IllustrationTitle.tsx

```diff
@@ -0,0 +1,20 @@
+import { Flex, Text } from "metabase/ui";
+
+interface IllustrationDescriptionProps {
+  title: string;
+  errorMessageContainerId: string;
+}
+
+export function IllustrationTitle({
```

Comment:
Now that we removed the info dot out this became a very simple component that displays the title with a placeholder for the error message.